Makefile

```
@@ -602,3 +602,127 @@ secretreader-plugin-image-build:
+		echo "✓ Found summary.yaml files"; \
+		find ./artifacts -name "summary.yaml" -exec echo "  - {}" \;; \
+	else \
+		echo "✗ No summary.yaml files found in artifacts"; \
+		exit 1; \
+	fi
+	@echo ""
+	@echo "To update thresholds based on these metrics, run:"
+	@echo "  make update-performance-thresholds"
+
+.PHONY: collect-performance-runs
+collect-performance-runs: ## Run performance test multiple times for threshold calculation
```

checkmake (0.3.2)

[warning] Target body for "collect-performance-runs" exceeds allowed length of 5 lines (18).

(maxbodylength)